File: Examples/Connect.cs
using System;
using System.Net;
using System.Net.FtpClient;

namespace Examples {
    public static class ConnectExample {
        public static void Connect() {
            using (FtpClient conn = new FtpClient()) {
                conn.Host = "localhost";
                conn.Credentials = new NetworkCredential("ftptest", "ftptest");
                conn.Connect();
            }
        }
    }
}

File: FluentFTP.VBExamples/Connect.vb
Imports System
Imports System.Net
Imports System.Threading
Imports System.Threading.Tasks
Imports FluentFTP

Namespace Examples
	Friend Module ConnectExample
		Sub Connect()
			Using conn = New FtpClient()
				conn.Host = "localhost"
				conn.Credentials = New NetworkCredential("ftptest", "ftptest")
				conn.Connect()
			End Using
		End Sub

		Sub ConnectAlt()
			Using conn = New FtpClient("127.0.0.1", "ftptest", "ftptest")
				conn.Connect()
			End Using
		End Sub

		Async Function ConnectAsync() As Task
			Dim token = New CancellationToken()

			Using conn = New AsyncFtpClient()
				conn.Host = "localhost"
				conn.Credentials = New NetworkCredential("ftptest", "ftptest")
				Await conn.Connect(token)
			End Using
		End Function

		Async Function ConnectAsyncAlt() As Task
			Dim token = New CancellationToken()

			Using conn = New AsyncFtpClient("127.0.0.1", "ftptest", "ftptest")
				Await conn.Connect(token)
			End Using
		End Function
	End Module
End Namespace
